conflict_resolution.html

```diff
@@ -0,0 +1,200 @@
+
+            <p>Chairs, ombudspeople, mediators, and investigators can recommend the following actions be taken if resolution is not possible:</p>
+
+            <ol>
+                <li>Corrective action. The individual who had the complaint raised against them will be advised to correct their behaviour going forward, for example not engaging in the same behaviours that resulted in the complaint.</li>
+                <li>Agree to a mediation process. The individual who had the complaint raised against them will be advised to participate in a mediation process, where both parties can agree to certain actions and changes going forward. </li>
+                <li>An official warning. In cases where the actions by an individual are frequent or severe, an individual and their employer (if relevant) will be given an official warning that their conduct is contrary to the CoC and that they should immediately desist in those behaviours. If they continue, this may result in the removal or suspension of their W3C participation. There is no time limit for the expiry of the warning.</li>
+                <li>Disciplinary meeting. In cases where the actions by an individual are frequent or severe, and where greater intervention is needed, an individual and a representative from their place of employment (if relevant) will be asked to attend a meeting with W3M. They will be told where their conduct is contrary to the CoC and that they should immediately desist in those behaviours. If they continue, they may result in the removal or suspension of their W3C participation.</li>
+                <li>Temporary removal from a working group, or as a chair of a working group or task force facilitator. In cases where the actions by an individual has been frequent and disruptive, or they have failed to adhere to actions agreed through mediation, they will be suspended from participating in any working groups, chair positions, or as a taskforce facilitator for a period of time agreed on by decision makers.</li>
+                <li>Final written warning or suspension. Where individuals have received multiple warnings, across the same or multiple parts of the CoC. The individual and their employer will receive a written warning that any further complaints will result in the removal of their W3C participation. They may also be suspended from W3C for a given length of time.</li>
+                <li>Termination of participation. Where there have been exceptional circumstances or frequent upheld complaints against an individual, they may have their participation in W3C terminated. The individual and their employer will receive a letter from the W3C outlining the reasons and why termination was warranted.</li>
```

Comment:
s/exceptional circumstances or frequent upheld complaints against an individual/egregious or repeated CoC violations/

s/they may have/the offender may have/

s/The individual/The offender/